python/ray/serve/handle.py

```diff
@@ -513,8 +513,9 @@ async def _to_object_ref_or_gen(
     def _to_object_ref_or_gen_sync(
         self,
         _record_telemetry: bool = True,
+        _allow_running_in_asyncio_loop: bool = False,
```

Comment:
Why do we need to add this parameter? When would it be set to `True`/`False`?

Reply:
disables the exception for internal usage